generator/sig_readme.tmpl

```diff
@@ -68,6 +68,18 @@ The following subprojects are owned by sig-{{.Label}}:
 {{- range .Owners }}
     - {{.}}
 {{- end }}
+{{- if .Contact.Slack }}
+  - Slack: [#{{.Contact.Slack}}](https://kubernetes.slack.com/messages/{{.Contact.Slack}})
+{{- end }}
+{{- if .Contact.MailingList }}
+  - Mailing List: [{{.Contact.MailingList}}]({{.Contact.MailingList}})
```

Comment:
Can we have this as `[Mailing list]({{.Contact.MailingList}})` to stay consistent with how the mailing list is rendered in other places? or else this would also display a huge URL. :grimacing:

Reply:
Something about Mailing List: Mailing List feels redundant to me but I can live and address that in followup

(tbh all of this bulleted formatting could stand to change but I'm not trying to refactor the world here)

Reply:
> Something about Mailing List: Mailing List feels redundant to me but I can live and address that in followup

I just meant one Mailing List with a link, like https://github.com/kubernetes/community/blob/master/sig-architecture/README.md#contact

Reply:
> (tbh all of this bulleted formatting could stand to change but I'm not trying to refactor the world here)

+1 to change and +1 to not-in-this-PR